src/nvim/version.h

```diff
@@ -11,20 +11,4 @@ extern char *longVersion;
 extern char *version_cflags;
 #endif
 
-//
```

Comment:
`version.c` will have "hardcoded" values instead and rely solely on vimpatch.lua.